cloud/index.rst

```diff
@@ -23,12 +23,125 @@ The key items you'll care about for each node:
+::
+
+  postgresql://[user[:password]@][host][:port][/dbname][?param1=value1&...]
+
+The connection string for each Cloud Formation is provided on the Overview tab in Citus Console.
+
+.. image:: ../images/cloud-overview-1.png
+
+By default the URL displays only the hostname of the connection, but the full URL is available by clicking the "Show Full URL" link.
+
+.. image:: ../images/cloud-overview-2.png
+
+For security Citus Cloud accepts only SSL connections, which is why the URL contains the :code:`?sslmode=require` parameter. To avoid a man-in-the-middle attack, you can also verify that the server certificate is correct. Download the official `Citus Cloud certificate <https://console.citusdata.com/citus.crt>`_ and refer to it in connection string parameters:
+
+::
+
+  ?sslrootcert=/location/to/citus.crt&sslmode=verify-full
+
+The string may need to be quoted in your shell to preserve the ampersand.
+
+.. note::
+
+  Database clients must support SSL to connect to Citus Cloud. In particular :code:`psql` needs to be compiled :code:`--with-openssl` if building PostgreSQL from source.
+
+A coordinator node on Citus Cloud has a hard limit of three hundred simultaneous active connections to limit memory consumption. If more connections are required, change the port in the connection URL from 5432 to 6432. This will connect to PgBouncer rather than directly to the coordinator, allowing up to roughly two thousand simultaneous connections. The coordinator can still only process three hundred at a time, but more can connect and PgBouncer will queue them.
+
+To measure the number of active connections at a given time, run:
+
+.. code-block:: postgresql
+
+  SELECT COUNT(*)
+    FROM pg_stat_activity
+   WHERE state <> 'idle';
+
+.. raw:: html
+
+Users and Permissions
+#####################
+
+As we saw above, every new Citus Cloud formation includes a user account called :code:`citus`. This account is great for creating tables and other DDL, but it has too much power for certain applications.
+
+We'll want to create new roles for specialized purposes. For instance, a user with read-only access is perfect for a web/reporting tool. The Cloud console allows us to create a new user, and will set up a new password automatically. Go to the "Roles" tab and click "Create New Role."
+
+.. image:: ../images/cloud-roles-tab.png
+
+It pops up a dialog where we will fill in the role name, which we can call :code:`reports`.
+
+.. image:: ../images/cloud-role-dialog.png
+
+After creating the role on a fresh formation, there will be three roles:
+
+.. code-block:: text
+
+  ┌─[ RECORD 1 ]────────────────────────────────────────────────────────────┐
+  │ Role name  │ citus                                                      │
+  │ Attributes │                                                            │
+  │ Member of  │ {reports}                                                  │
```

Comment:
It's a role that cloud creates for each formation, and uses in the connection string it provides. The role has enough permissions to create db objects, but it's not a superuser.